python/requirements.txt

```diff
@@ -43,6 +43,8 @@ starlette
 
 # Requirements for running tests
 blist; platform_system != "Windows"
+azure-common
+azure-mgmt-resource
```

Comment:
why do we need these?

Reply:
Those are required when we import from `from ray.autoscaler._private.azure.config import (_configure_key_pair as _azure_configure_key_pair)`